src/frontend/src/App.tsx

```diff
@@ -1,5 +1,4 @@
 import React, { Component } from 'react';
-import './App.css';
```

Comment:
This causes an error when running jest since can't parse the css, but without this line, the CSS isn't used when run the app...

Reply:
Fixed by bb8599edd7782ad260462dce99485e241654c352